helpers/fetchCampaignBySlugServerOnly.js

```diff
@@ -0,0 +1,18 @@
+import gpApi from 'gpApi';
+import { getServerToken } from 'helpers/userServerHelper';
+import gpFetch from 'gpApi/gpFetch';
+
+export async function fetchCampaignBySlugServerOnly(slug) {
+  // admin only
+  try {
+    const api = gpApi.campaign.findBySlug;
+    const payload = {
+      slug,
```

Comment:
same as above. This is admin only. Probably belongs under admin/shared instead of /shared